setup.py

```diff
@@ -237,7 +237,7 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
 kubernetes = ['kubernetes>=3.0.0',
               'cryptography>=2.0.0']
 ldap = ['ldap3>=2.5.1']
-mssql = ['pymssql>=2.1.1']
+mssql = ['pymssql<3.0']
```

Comment:
```suggestion
mssql = ['pymssql>=2.1.1,<3.0']
```

Reply:
Or 
```suggestion
mssql = ['pymssql~=2.1.1']
```

Reply:
Yep.  ~= 2.1.1 should work as equivalent of 2.1.*